ptclient/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto eol=lf
```

Comment:
Ensures correct line endings when committing to repo. Downstream developers benefit from this where their IDE settings may be set to CRLF or auto.